spec/concepts/stock/task/truncate_table_spec.rb

```diff
@@ -0,0 +1,53 @@
+require 'rails_helper'
+
+describe Stock::Task::TruncateTable do
+  subject { described_class.call table_name: 'etablissements', logger: logger }
+
+  let(:logger) { instance_spy Logger }
+  let(:siren) { '005880034' }
+
+  before do
```

Comment:
Le `before` ici s'applique une seule fois avant le context n'est ce pas ? Je ne comprend pas pourquoi ces tests fonctionnent du coup, je pensais que chaque `subject` dans les `it` executais notre `described_class.call`, du coup la classe serait appellée trois fois non ? J'ai probablement tord mais je ne sais pas où

Reply:
chaque test est lancé dans une transaction SQL, ensuite on lance le `before`, puis le `it` et ensuite ça rollback.

Pas sûr d'avoir répondu à la question

Reply:
Je suis pas certain de ce que tu veux dire par "le test est lancé dans une transaction SQL", je vais me renseigner la dessus